public/v4/apps/meilisearch.yml

```diff
@@ -0,0 +1,25 @@
+captainVersion: 4
+services:
+    $$cap_appname:
+        image: getmeili/meilisearch:$$cap_app_version
+        volumes:
+            - $$cap_appname-data-ms:/data.ms
+        restart: always
+        caproverExtra:
+            containerHttpPort: '7700'
+caproverOneClickApp:
+    variables:
```

Comment:
do we need the MEILI_MASTER_KEY to connect ?

Reply:
I am working on adding all the environment variables